hphp/runtime/vm/jit/code-cache.cpp

```diff
@@ -97,7 +98,14 @@ CodeCache::CodeCache()
     exit(1);
```

Comment:
@mxw - Hi Max, et.al,  In my previous life we called those trampolines branch islands.  The linker would create a little table between object files for the case you describe above.  Is there an easy way to instrument this an capture some data?  It sure seems like most of the jumps are nearby, but that could be an artifact of just looking at small-ish tests.